### PR TITLE
Update Advantech esrp uno interface and model

### PR DESCRIPTION
**Pull request by Harvey-Tsai**
### Company Info

Advantech 

Company website: www.advantech.com
GitHub : https://github.com/Harvey-Tsai/iot-plugandplay-models

### IoT Plug and Play Device Info

ESRP-CSS-UNO1372: https://www.advantech.com/products/05dd6ca3-f058-4c73-824e-dc9143898a45/esrp-css-uno1372/mod_0fa58890-1962-4901-83b9-1e08eada3bb3
OS & Arch : Ubuntu 18.04 & x86_64

ESRP-CSS-UNO2271: https://www.advantech.com/products/05dd6ca3-f058-4c73-824e-dc9143898a45/esrp-css-uno2271/mod_c0ddbe2f-5c75-486d-b2e4-312d89c04649
OS & Arch : Ubuntu 18.04 & x86_64

ESRP-CSS-UNO2372: https://www.advantech.com/products/05dd6ca3-f058-4c73-824e-dc9143898a45/esrp-css-uno2372/mod_0ea177e9-c4a7-44c6-9342-a026cecc871f
OS & Arch : Ubuntu 18.04 & x86_64

ESRP-CSS-UNO2484: https://www.advantech.com/products/05dd6ca3-f058-4c73-824e-dc9143898a45/esrp-css-uno2484/mod_98d0b774-c73f-4b70-9f79-919e7fc24ce0
OS & Arch : Ubuntu 18.04 & x86_64

### Model Submission Goals

- Device certification
- Presence in the [Certified Device catalog](https://devicecatalog.azure.com/)
- IoT Central integration
- Custom solution

### Code Owners & Reserved Names

Code Owners : Harvey.Tsai
mail: Harvey.Tsai@advantech.com.tw

### CODE REVIEW

File: dtmi/advantech/esrp_uno1372-2.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "@id": "dtmi:Advantech:esrp_uno1372;2",
+  "@type": "Interface",
+  "displayName": "ADVANTECH device Model",
```
Comment (rido-min) [inline dtmi/advantech/esrp_uno1372-2.json:4]: provide a more descriptive `displayName`

File: dtmi/advantech/esrp_uno2484-2.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "@id": "dtmi:Advantech:esrp_uno2484;2",
+  "@type": "Interface",
+  "displayName": "ADVANTECH device Model",
```
Comment (rido-min) [inline dtmi/advantech/esrp_uno2484-2.json:4]: provide a more descriptive `displayName`

File: dtmi/advantech/esrp_uno2271-2.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "@id": "dtmi:Advantech:esrp_uno2271;2",
+  "@type": "Interface",
+  "displayName": "ADVANTECH device Model",
```
Comment (rido-min) [inline dtmi/advantech/esrp_uno2271-2.json:4]: provide a more descriptive `displayName`

**Review by rido-min — CHANGES_REQUESTED** [review]
provide a more descriptive `displayName` for each interface

File: dtmi/advantech/esrp_uno1372-2.json (hunk 1 above)
Comment (Harvey-Tsai) [reply dtmi/advantech/esrp_uno1372-2.json:4]: "displayName": "ESRP_UNO1372",

File: dtmi/advantech/esrp_uno2484-2.json (hunk 2 above)
Comment (Harvey-Tsai) [reply dtmi/advantech/esrp_uno2484-2.json:4]: "displayName": "ESRP_UNO2484",

File: dtmi/advantech/esrp_uno2271-2.json (hunk 3 above)
Comment (Harvey-Tsai) [reply dtmi/advantech/esrp_uno2271-2.json:4]: "displayName": "ESRP_UNO2271",

**Review by rido-min — APPROVED** [review]

**Pull request merged**